### PR TITLE
prefix testsuite name to testnames when there is no wrapping testsuites element

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: prow/process.go [hunk 1]
```diff
@@ -326,7 +326,14 @@ func (a *LogAccumulator) AddSuites(ctx context.Context, suites junit.Suites) {
 			case test.Error != nil:
 				out = *test.Error
 			}
-			fmt.Fprintf(f, "\n\n# %s\n", test.Name)
+			// testgrid prefixes testnames with their suitename, if the junit xml they came from
+			// has a wrapping <testsuites> element.  So keep the testnames aligned
+			// by prefixing here as well.
```
Comment (smarterclayton) [inline prow/process.go:331]: I'm going to ask the question here - is testgrid the one who is broken (by picking an arbitrary constraint that doesn't follow a sensible rule)?

Comment (bparees) [reply prow/process.go:331]: i don't know, i can squint and see a rationale behind their choice (if you've only got one suite, the suitename is not that interesting, if you have a set of suites, it probably is).  So i wouldn't characterize it as "an arbitrary constraint that doesn't follow a sensible rule."  Would i have done it differently (always or never prefix)?  yes.   Do i expect they're going to change it?  no.

here's my discussion with Michelle:
https://kubernetes.slack.com/archives/C09QZ4DQB/p1598378171034300?thread_ts=1598360817.034000&cid=C09QZ4DQB

**Pull request merged**